107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -89,8 +89,9 @@ public void mergeConfigNoTemplateNoLoaderWriter() {
     boolean storeByValue = false;
     Collection<ServiceConfiguration<?>> serviceConfigurations = configHolder.cacheConfiguration.getServiceConfigurations();
     for (ServiceConfiguration<?> serviceConfiguration : serviceConfigurations) {
-      if (serviceConfiguration instanceof OnHeapStoreServiceConfiguration) {
-        storeByValue = ((OnHeapStoreServiceConfiguration) serviceConfiguration).storeByValue();
+      if (serviceConfiguration instanceof CopierConfiguration) {
+        storeByValue = true;
```

Comment:
@ljacomet I'm not quite clear on what you meant here. The old code was checking if a cache is store-by-value based on the storeByValue field of the OnHeapStoreServiceConfiguration if one such config is found. But with my changes it is checked on the basis of the presence of an explicit CopierConfiguration. So I don't understand why that fails this test.